### PR TITLE
07 29  2/x add transaction for enabling unclaimed token transfer permissions

### PR DESCRIPTION
**Pull request by jmilldotdev**
## What does this PR do and why?

Adds veNFT section to project tools drawer, moves veNFT creator modal, and adds button which adds Operator permissions to the veNFT contract to transfer unclaimed project tokens

## Screenshots or screen recordings

_If applicable, provide screenshots or screen recordings to demonstrate the changes._

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

File: src/components/veNft/VeNftSetUnclaimedTokensPermissionSection.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        onClick={unclaimedTokensPermission}
+        loading={loading}
+      >
+        Enable
+      </Button>
+    </section>
+  )
+}
+
+export default VeNftSetUnclaimedTokensPermissionSection
```
Comment (wraeth-eth) [inline src/components/veNft/VeNftSetUnclaimedTokensPermissionSection.tsx:52]: **question**: Is this the goto way to frame default exports? I have seen a couple of repos like this. Should probably start doing it too

**Review by wraeth-eth — COMMENTED** [review]
Looking good jmil - very minor comments

File: src/hooks/veNft/transactor/VeNftUnclaimedTokenPermissions.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+import { V2ProjectContext } from 'contexts/v2/projectContext'
+
+import { VENFT_CONTRACT_ADDRESS } from 'constants/veNft/veNftProject'
+
+export function useUnclaimedTokensPermissionTx(): TransactorInstance {
+  const { transactor, contracts } = useContext(V2UserContext)
+  const { projectId } = useContext(V2ProjectContext)
+  const contractAddress = VENFT_CONTRACT_ADDRESS
+  const permissionIndexes = [12]
```
[the remark this answers is not in the document]
Comment (jmilldotdev) [reply src/hooks/veNft/transactor/VeNftUnclaimedTokenPermissions.ts:14]: true

File: src/components/veNft/VeNftSetUnclaimedTokensPermissionSection.tsx (hunk 1 above)
Comment (jmilldotdev) [reply src/components/veNft/VeNftSetUnclaimedTokensPermissionSection.tsx:52]: i have a snippet in vscode where i type `rafce` and it scaffolds a component like this. i don't know if there's a right or wrong way but that's just my muscle memory for new component files

**Pull request merged**